packages/calypso-analytics/src/index.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * External dependencies
+ */
+import { assign, includes, isObjectLike, isUndefined, omit, pickBy, times } from 'lodash';
+import { parse } from 'qs';
+import cookie from 'cookie';
+import url from 'url';
+import debug from 'debug';
+import { EventEmitter } from 'events';
+
+/**
+ * Internal Dependencies
+ */
+
+import { getCurrentUser, setCurrentUser } from './utils/current-user';
+import { loadScript } from '@automattic/load-script';
+import doNotTrack from './utils/do-not-track';
+
+/**
+ * Re-export
+ */
+export { default as doNotTrack } from './utils/do-not-track';
+export { getCurrentUser, setCurrentUser } from './utils/current-user';
```

Comment:
It took me a while to figure out how `default as doNotTrack` was working. Do you think it'd be clearer to replace these two lines with a single one:

```js
export { doNotTrack, getCurrentUser, setCurrentUser };
```